### DISCUSSION TITLE
Reverting flow editor causes the Simulator tab to disappear

### DISCUSSION THREAD

**Discussion by colenewcomer** [nyaruka/floweditor#1163] [Ideas]
RePro:
-Attempt to revert a flow to its previous state

Expected: the page loads as it should, keeping elements
Actual: The flow editor is cleared and is no longer accessible
No console error

**Comment by ericnewcomer** [comment]
I'm unable to repro this. Can you provide more details? There must be more to it. Can you include the console error?

**Comment by colenewcomer** [comment]
There is no console error. If you make a flow with a few steps and the revert it back the flow tester disappears from the side of the screen

**Comment by ericnewcomer** [comment]
Maybe provide a screencast of the repro? I'm able to open flows and revert them without issue.

**Comment by colenewcomer** [comment]
I intended to say the flow simulator, apologies, I misspoke.

**Comment by ericnewcomer** [comment]
Can confirm.

When reverting the flow version, after reverting the simulator tab disappears. When reloading the page, it returns.
